docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,11 +6,14 @@
 
 ## Checklist:
 
-_these will eventually be enforced by CircleCI; we need to set that up at the USDOJ org level_
+### Author
 
-+ [ ] If front end or functionality change, run locally and check http://0.0.0.0:8000/report/.
-+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests)
```

Comment:
Removing this line because the CI will now tell us if tests are passing and failing; no manual check from devs required.

Reply:
We might want to keep the test passing line. We are not programmatically forcing the tests to pass for merging. Though, the tests should be passing before requesting a review.

Reply:
Oh, interesting! I'd like to ask Darren to enforce required status checks against `develop` and `master`. Release (staging) might be a little more complex because it uses a regex rather than a constant branch name.

Reply:
Yea, let's add that back to be clear that is part of our review process.

Reply:
Done!